src/stores/manager/keys/hashicorp.go

```diff
@@ -22,6 +22,10 @@ func NewHashicorpKeyStore(specs *entities.HashicorpSpecs, logger log.Logger) (*h
 		return nil, errors.ConfigError(errMessage)
 	}
 
+	if cfg.SkipVerify {
```

Comment:
@sylvain.decourval you need to add this code to the function Create in src/stores/connectors/stores/create.go instead of where it's now. Should be added in both switches of HashicorpKeys and HashicorpSecrets.
The check will be by checking the specs spec.SkipVerify